plugins/MauticCrmBundle/Integration/SalesforceIntegration.php

```diff
@@ -2032,11 +2032,15 @@ protected function processCompositeResponse($response, &$totalUpdated = 0, &$tot
                     $this->logIntegrationError($exception);
                     $integrationEntity = null;
                     if ($integrationEntityId && $object !== 'CampaignMember') {
-                        $integrationEntity = $this->em->getReference('MauticPluginBundle:IntegrationEntity', $integrationEntityId);
-                        $integrationEntity->setLastSyncDate(new \DateTime());
+                        $integrationEntity = $this->em->find('MauticPluginBundle:IntegrationEntity', $integrationEntityId);
```

Comment:
I do not like this solution. Why not to use any Model class or repository at least?